pkg/lib/feature/siwe/errors.go

```diff
@@ -5,5 +5,7 @@ import (
 )
 
 var InvalidNonce = apierrors.Forbidden.WithReason("InvalidNonce")
+var InvalidNetwork = apierrors.BadRequest.WithReason("InvalidNetwork")
```

Comment:
Construct this error with details. The details should contain the expected network. In UI, tell the user the expected network (both the numeric ID and the name)